packages/components/cascader-panel/src/node.vue

```diff
@@ -25,14 +25,14 @@
       :indeterminate="node.indeterminate"
       :disabled="isDisabled"
       @click.stop
-      @update:model-value="handleCheck"
+      @update:model-value="handleSelectCheck"
```

Comment:
If the checkbox execute the same method, we should consider the value of checkStrictly?

Reply:
yes, i add a if-else